app/Models/Report.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Report extends Model
+{
+    use SoftDeletes;
+    /**
+     * [$table description]
+     * @var string
+     */
+    protected $table= 'reports';
+    /**
+     * [$dates description]
+     * @var array
+     */
+    protected $dates = ['deleted_at'];
+    /**
+     * [$fillable description]
+     * @var [type]
+     */
+    protected $fillable = [
+        'user_id',
+        'post_id',
+        'type',
+        'created_at',
+        'updated_at',
+        'deleted_at'
+    ];
+    /**
+     * Get users: One to many
+     * @return [type] [description]
+     */
+    public function users() {
+        return $this->belongsTo('App\Models\User', 'user_id');
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Models/Report.php</br>Opening brace should be on a new line